README.md

```diff
@@ -69,3 +69,53 @@ app.post('/contacts/:contactid', function (req, res) {
 	});
 });
 ```
+
+### Proposed item function reference
+Via the use of optional parameters and dedicated functions it is possible to retrieve anything from a single field on a single record to all fields on all records. Selection criteria and response timeouts may also be specified.
+
+The criteria parameter defines the query string to use to restrict the number of records that are returned. It followes the normal name=value&name=value... syntax
```

Comment:
I think it'd be tidier to pass in an object of key/value pairs and do the calls to `remove_blank_values` and `querystring.stringify` inside this library.